packages/blocks/src/components/InlineBanner/InlineBanner.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+type InlineBannerProps = {
+  onClick?: () => void;
+};
+
+export const InlineBanner: React.FC<InlineBannerProps> = ({ onClick }) => {
+  return (
+    <div className="t-inline-banner">
+      <p>
+        Want to automatically add repos from this organization?{' '}
+        <a
+          onClick={() => {
+            if (onClick) {
+              onClick();
+            }
+          }}
+        >
+          {' '}
+          Auto import
+        </a>{' '}
+        repos from organization instead
+      </p>
+    </div>
```

Comment:
Contents of this should be passable via children - the component will just wrap the base styles for the banner